api/go.mod

```diff
@@ -0,0 +1,9 @@
+module api
```

Comment:
While this api name will work fine for this bootcamp. Usually the naming convention for your projects with go mod would be like how you would see on external dependencies `github.com/<ownerOfTheProject>/<projectName>`